packages/engine/crawlers/solana/index.ts

```diff
@@ -56,10 +59,14 @@ export const solanaEngineRunner: SolanaRunner = {
 				item.account.quotes = quotes[makeHashId(item)].quotes;
 			}
 
-			tokenActions.setItems(fungibleTokens);
+			tokenActions.setTokens(fungibleTokens);
+			solanaCollectiblesByAddress({
```

Comment:
use `Promise.all` to wait for collectibles setup?

Reply:
already refactored and used `Promise.all` in `setTokens` method